pyartifactory/models/repository.py

```diff
@@ -13,6 +13,7 @@ class PackageTypeEnum(str, Enum):
     """Enumerates package types."""
 
     alpine = "alpine"
+    ansible = "ansible"

```

Comment:
Ansible package type is missing from the [Artifactiory API ](https://jfrog.com/help/r/jfrog-rest-apis/repository-configuration-json). Can you please remove it?

Reply:
I see it is not in there, but it is an option when creating a repository in the interface. 
Interestingly enough this works with the changes applied and the repository is created with the ansible package type as expected:

```
from pyartifactory import Artifactory
from pyartifactory.models.repository import RemoteRepository
art = Artifactory(
    url="http://docker-compose-artifactory.local:8082/artifactory", 
    auth=("*******","********"), 
    cert=None,
    api_version=2,
    verify=False
)
repo_created = art.repositories.create_repo(RemoteRepository(
    packageType="ansible",
    url="https://galaxy.ansible.com",
    key="testrepo-ansible"    
))
print(repo_created)
repo_req = art.repositories.get_repo("testrepo-ansible")
print(repo_req)

```
(Artifactory 7.90.5 rev 79005900)

Reply:
Yeah I see. I'm approving the PR, but can you open a support ticket for them to add ansible to their public documentation ?

Reply:
Thank you, and done: [RTFACT-30747](https://jfrog.atlassian.net/browse/RTFACT-30747)